src/System.Text.Json/tests/JsonDocumentTests.cs

```diff
@@ -3034,6 +3045,124 @@ public static void EnsureResizeSucceeds()
             }
         }
 
+        [Fact]
+        public static void ValueEquals_Null_True()
+        {
+            const string jsonString = "   null   ";
+            JsonElement jElement = (JsonElement)JsonDocument.Parse(jsonString).RootElement.Clone();
```

Comment:
This should be

```C#
using (JsonDocument doc = JsonDocument.Parse(jsonString))
{
    JsonElement jElement = doc.RootElement;

    // tests go here
}
```

The construction here is maximally memory inefficient (rents from the array pool, never returns it, and makes an extra copy)